provider/http.go

```diff
@@ -151,6 +151,19 @@ func (h *HTTPClient) GetOpenPerpOrders(ctx context.Context, request *pb.GetOpenP
 	return orders, nil
 }
 
+// GetDriftOpenMarginOrders returns all opened margin orders on Drift platform
+func (h *HTTPClient) GetDriftOpenMarginOrders(ctx context.Context, request *pb.GetDriftOpenMarginOrdersRequest) (*pb.GetDriftOpenMarginOrdersResponse, error) {
+	marketsString := convertStrSliceArgument("markets", false, request.Markets)
+	url := fmt.Sprintf("%s/api/v2/drift/margin-open-orders?ownerAddress=%s&accountAddress=%s&%s",
```

Comment:
i think you might be missing an argument at the end of this one? `&%s`

Reply:
removed & which is extra